app/assets/stylesheets/govuk_publishing_components/components/govspeak/_content-block.scss

```diff
@@ -0,0 +1,32 @@
+@import "govuk/core/links";
+@import "govuk/core/lists";
+@import "govuk/core/typography";
+
+@import "content_blocks/contact";
+
+.govspeak, // Legacy class name that's still used in some content items - needs to be kept until `.govspeak` is removed from the content items.
+.gem-c-govspeak {
+  .content-block {
+    .content-block__body {
+      @extend %govuk-body-m;
```

Comment:
Curious about the logic behind these class definitions (this one and `.content-block__link` on line 28). If you can control the class names in the code and they're only extending existing `govuk-frontend` classes, why not use those classes directly in the code i.e. `govuk-body` and `govuk-link`?

Reply:
(same for contact block title and subtitle)